src/worker.js

```diff
@@ -92,6 +92,7 @@ const initServer = (app, config, callback) => {
   runAppLifecycle('initialize', config.plugins, config)
     .then(() => {
       server = app.listen(config.port, callback);
+      return server;
```

Comment:
In that case, we need a `.catch` handler on the `app.listen` call, no? (assuming it returns a promise)